pos_localstorage/static/src/js/models.js

```diff
@@ -0,0 +1,224 @@
+openerp.pos_localstorage = function(instance){
+    var module = instance.point_of_sale;
+
+    var UpdateProducts = {
+        runOnce: function(pos) {
+            var self = this;
+            // We subtract the following period from the last update time
+            //  to make sure that we get all changes (including those that happened
+            //  during the last update phase).
+            var _searchGracePeriod = 30 * 60 * 1000;  // 30 minutes
+            // List of product attributes that the POS uses.
+            var productAttributeList = ['name', 'list_price','price','pos_categ_id', 'taxes_id', 'ean13', 'default_code', 'variants',
+                     'to_weight', 'uom_id', 'uos_id', 'uos_coeff', 'mes_type', 'description_sale', 'description'];
+
```

Comment:
Could you set searchGracePeriod and productAttributeList as model vars to allow other modules to overload more easely ?